tests/conftest.py

```diff
@@ -333,6 +325,29 @@ def check_ready(self, timeout=30):
 
             time.sleep(1)
 
+    # FIXME: For the sake of stabilizing the PR pipeline this method currently assumes 32 ports
+    # (much like the rest of the test suite). This should be adjusted to accomodate a dynamic
+    # number of ports. GitHub Issue: Azure/sonic-swss#1384.
+    def check_swss_ready(self, timeout=60):
+        """Verify that SWSS is ready to receive inputs.
+
+        Almost every part of orchagent depends on ports being created and initialized
+        before they can proceed with their processing. If we start the tests after orchagent
+        has started running but before it has had time to initialize all the ports, then the
+        first several tests will fail.
+        """
+        num_ports = 32
```

Comment:
@daall We can define macro for 32 (few places we are using this)

Reply:
True, added!